### PR TITLE
Support adding custom client scripts to the tested page

### PR DESCRIPTION
**Pull request by miherlosev**
Changes:
* Router.unregisterRoute(route: string, method: string)
* `new RequestInfo(ctx, ctx.reqOpts);` -> `new RequestInfo(ctx);`
* remove 'use strict' for the server test files
* add `const PAGE_ACCEPT_HEADER=...` (used in tests)
* move service route paths to separate file (`src/proxy/service-routes.ts`)

### CODE REVIEW

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

File: src/request-pipeline/header-transforms/index.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ function transformHeaders (srcHeaders, ctx, transformList, forcedTransforms) {
 }
 
 // API
-export function forRequest (ctx): { [name: string]: string|Array<string> } {
+export function forRequest (ctx): { [name: string]: string } {
```
Comment (LavrovArtem) [inline src/request-pipeline/header-transforms/index.ts]: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/http.d.ts#L8

File: src/request-pipeline/context.ts [hunk 2]
```diff
@@ -272,9 +274,17 @@ export default class RequestPipelineContext {
...
+        return this.session.injectable.userScripts
+            .filter(userScript => userScript.page.match(requestInfo))
+            .map(userScript => userScript.url);
+    }
+
     getInjectableScripts (): Array<string> {
-        const taskScript = this.isIframe ? '/iframe-task.js' : '/task.js';
-        const scripts    = this.session.injectable.scripts.concat(taskScript);
+        const taskScript = this.isIframe ? SERVICE_ROUTES.iframeTask : SERVICE_ROUTES.task;
+        const scripts    = [].concat(this.session.injectable.scripts, taskScript, this._getInjectableUserScripts());
```
Comment (LavrovArtem) [inline src/request-pipeline/context.ts]:
```js
const scripts    = this.session.injectable.scripts.concat(taskScript, this._getInjectableUserScripts());
```

**Pull request merged**